src/server.c

```diff
@@ -635,6 +644,15 @@ dictType hashDictType = {
     dictSdsDestructorM          /* val destructor */
 };
 
+dictType hashDictTypeZ = {
```

Comment:
add description of this structure, like for example it is done for  _commandTableDictType_
/* Command table. sds string -> command struct pointer. *